Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -94,7 +94,7 @@ void SettingsWindow::AddCategoryToList(const QString& title, const std::string&
 void SettingsWindow::MakeCategoryList()
 {
   m_categories = new QListWidget;
-  m_categories->setMaximumWidth(175);
+  m_categories->setFixedWidth(125);
```

Comment:
Is your intent to give the list widget a minimal-ish width? If so, something like

```c++
m_categories->setFixedWidth(m_categories->sizeHintForColumn(0) + m_categories->verticalScrollBar()->sizeHint().width() + 2 * m_categories->frameWidth());
```

is probably more future-proof (works on macOS) (do this after adding categories, and removing the manual QListWidgetItem size hint in `AddCategoryToList`).